src/App/components/Checkbox.js

```diff
@@ -0,0 +1,33 @@
+import React, { memo } from 'react'
+import styled from 'styled-components'
+
+const CheckBox = styled.input`  
+  height: 40px;
+  background: none;
+  opacity: 0;
+  text-align: center;
+  width: 40px;
+  height: auto;
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  margin: auto 0;
+  border: none;
+  -webkit-appearance: none;
+  appearance: none;
+
+  &:checked + .Data {
+    background-image: url('data:image/svg+xml;utf8,%3Csvg%20xmlns%3D%22http%3A//www.w3.org/2000/svg%22%20width%3D%2240%22%20height%3D%2240%22%20viewBox%3D%22-10%20-18%20100%20135%22%3E%3Ccircle%20cx%3D%2250%22%20cy%3D%2250%22%20r%3D%2250%22%20fill%3D%22none%22%20stroke%3D%22%23bddad5%22%20stroke-width%3D%223%22/%3E%3Cpath%20fill%3D%22%235dc2af%22%20d%3D%22M72%2025L42%2071%2027%2056l-4%204%2020%2020%2034-52z%22/%3E%3C/svg%3E');
```

Comment:
What is `.Data`?

Reply:
this .Data is stand for the JSX element DataLabel, I renamed it to .DataLabel to make it more clearly

Reply:
If you want to style DataLabel, please use styled-components for it. Do not code this way.